<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* General Window */
QWidget {
    background-color:rgb(255, 255, 255); /* Light Gray for Light Mode */
    color: #000000;           /* Black Text */
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
    font-size: 12pt;
border-radius: 10px;
}

/* Dark Mode (Toggle this for dark theme) */
/*
QWidget {
    background-color: #2b2b2b; 
    color: #ffffff;           
}
*/

/* Buttons */
QPushButton {
    background-color: #0078d7; /* Windows Blue */
    color: white;
    border-radius: 5px;
    padding: 8px 16px;
    border: none;
    font-size: 11pt;
}
QPushButton:hover {
    background-color: #005a9e; /* Darker Blue on Hover */
}
QPushButton:pressed {
    background-color: #004578; /* Even Darker Blue on Click */
}

/* Input Fields */
QLineEdit {
    background-color: white;
    color: black;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
}
QLineEdit:focus {
    border: 1px solid #0078d7; /* Blue Border on Focus */
    background-color: #ffffff;
}

/* Labels */
QLabel {
    font-size: 13pt;
    color: #333333; /* Dark Gray */
}

/* Checkboxes and Radio Buttons */
QCheckBox, QRadioButton {
    spacing: 6px;
    color: #333333;
}

/* Scrollbars */
QScrollBar:vertical {
    border: none;
    background: #f3f3f3;
    width: 8px;
    margin: 0px 0px 0px 0px;
}
QScrollBar::handle:vertical {
    background: #cccccc;
    border-radius: 4px;
}
QScrollBar::handle:vertical:hover {
    background: #aaaaaa;
}
</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>381</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="email_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>381</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>phone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>550</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="add_student_button">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Add Student</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/* Cancel Button */




/*#bababa*/

/*background-color: transparent; */



QPushButton {
    background-color: #bababa; /* No background */
    color: black;                  /* Black text */
    border: 1px solid #cccccc;     /* Light gray border */
    border-radius: 5px;
    padding: 8px 16px;
    font-size: 10pt;
}

QPushButton:hover {
    background-color: #e5e5e5; /* Light gray on hover */
    border: 1px solid #bbbbbb; /* Slightly darker gray border on hover */
}

QPushButton:pressed {
    background-color: #d9d9d9; /* Darker gray on press */
    border: 1px solid #aaaaaa; /* Even darker gray border on press */
}
</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>211</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>last name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="last_name_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="classes_dropdown">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">            QComboBox {
                background-color: #0078d7;  /* Blue background */
                color: white;              /* White text */
                border: none;
                border-radius: 5px;        /* Rounded corners */
                font-size: 10pt;
            }
            QComboBox:hover {
                background-color: #005a9e; /* Darker blue on hover */
            }
 


            QComboBox QAbstractItemView {
                background-color: #bababa; /* White background for drop-down menu */
                color: black;              /* Black text in drop-down */

            }



            QComboBox::down-arrow {
                width: 8px;
                height: 8px;
                margin: 0px;
            }


           QComboBox::drop-down {
                width: 25px;

            }
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>class</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">            QComboBox {
                background-color: #0078d7;  /* Blue background */
                color: white;              /* White text */
                border: none;
                border-radius: 5px;        /* Rounded corners */
                font-size: 10pt;
            }
            QComboBox:hover {
                background-color: #005a9e; /* Darker blue on hover */
            }
 


            QComboBox QAbstractItemView {
                background-color: #bababa; /* White background for drop-down menu */
                color: black;              /* Black text in drop-down */

            }



            QComboBox::down-arrow {
                width: 8px;
                height: 8px;
                margin: 0px;
            }


           QComboBox::drop-down {
                width: 25px;

            }
</string>
   </property>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="birth_date">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    background-color: white;
    color: black;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1999</year>
     <month>12</month>
     <day>25</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>d/M/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>381</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>address</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="address_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add_pic">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>add pic</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    spacing: 6px; /* Space between the radio button and label text */
    color: black; /* Text color */
    font-size: 11pt;
}

QRadioButton::indicator {
    width: 14px;
    height: 14px;
    border-radius: 9px; /* Makes the indicator circular */
    border: 2px solid #0078D4; /* Fluent Blue border */
    background-color: transparent; /* Transparent background */
}

QRadioButton::indicator:hover {
    border-color: #005a9e; /* Darker Fluent Blue on hover */
}

QRadioButton::indicator:checked {
    background-color: #0078D4; /* Fluent Blue when checked */
    border-color: #0078D4; /* Fluent Blue border */
}

QRadioButton::indicator:pressed {
    background-color: #004578; /* Darker blue on press */
}

QRadioButton::indicator:checked:hover {
    background-color: #005a9e; /* Slightly darker blue on hover when checked */
}

QRadioButton::indicator:disabled {
    border-color: #d6d6d6; /* Light gray for disabled state */
    background-color: #f0f0f0; /* Light gray background for disabled */
}

QRadioButton::indicator:checked:disabled {
    background-color: #d6d6d6; /* Gray checked state for disabled */
    border-color: #d6d6d6;
}
</string>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton {
    spacing: 6px; /* Space between the radio button and label text */
    color: black; /* Text color */
    font-size: 11pt;
}

QRadioButton::indicator {
    width: 14px;
    height: 14px;
    border-radius: 9px; /* Makes the indicator circular */
    border: 2px solid #0078D4; /* Fluent Blue border */
    background-color: transparent; /* Transparent background */
}

QRadioButton::indicator:hover {
    border-color: #005a9e; /* Darker Fluent Blue on hover */
}

QRadioButton::indicator:checked {
    background-color: #0078D4; /* Fluent Blue when checked */
    border-color: #0078D4; /* Fluent Blue border */
}

QRadioButton::indicator:pressed {
    background-color: #004578; /* Darker blue on press */
}

QRadioButton::indicator:checked:hover {
    background-color: #005a9e; /* Slightly darker blue on hover when checked */
}

QRadioButton::indicator:disabled {
    border-color: #d6d6d6; /* Light gray for disabled state */
    background-color: #f0f0f0; /* Light gray background for disabled */
}

QRadioButton::indicator:checked:disabled {
    background-color: #d6d6d6; /* Gray checked state for disabled */
    border-color: #d6d6d6;
}
</string>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(87, 227, 137);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
